source/Ship.cpp

```diff
@@ -2146,7 +2147,8 @@ double Ship::IdleHeat() const
 	// heat * (1 - diss + activeCool / (100 * mass)) = (heatGen - cool)
 	double production = max(0., attributes.Get("heat generation") - cooling);
 	double dissipation = .001 * attributes.Get("heat dissipation") + activeCooling / (100. * Mass());
-	return production / dissipation;
+
+	return dissipation != 0. ? production / dissipation : numeric_limits<double>::infinity();
```

Comment:
If there is no dissipation and no production, then the ship's IdleHeat is its current heat, not infinity.
If there is no dissipation and any production, then the ship's IdleHeat is +infinity

Reply:
@tehhowch Why with both null, does it return the current heat? The heat at idle doesn't depend on the current heat, no?

Reply:
the idle heat is whatever heat it has where its production is exactly offset by its dissipation. For 0 and 0, that is every possible value of heat.

Reply:
@tehhowch Ohh. Now it makes sense! :) Thank you